tests/integration/docker/volume.bats

```diff
@@ -48,6 +48,31 @@ setup() {
 	[ $RC -eq 0 ]
 }
 
+@test "Volume - bind-mount a directory" {
+	dir_path="$(pwd)/sharedSpace"
+	test_file="foo"
+	containerPath="/root/sharedSpace"
+	mkdir "$dir_path"
```

Comment:
Yes, that works too :) I'd suggest adding a comment in the test explaining why you are delaying the test on `$status`.